src/components/ReviewComponent.jsx

```diff
@@ -21,18 +21,18 @@ class ReviewComponent extends Component {
       destination: destination,
       sender: sender,
       password: password,
-      gasCost: gasCost
+      txCost: txCost
     })
   }
 
   componentDidMount () {
     // refresh gas cost
-    const { cryptoSelection } = this.props
-    this.props.getGasCost({ cryptoType: cryptoSelection })
+    const { cryptoSelection, getTxCost } = this.props
+    getTxCost({ cryptoType: cryptoSelection })
```

Comment:
do we have sufficient params for getTxCost? see https://github.com/ventureum/Chainsfer/blob/fa1cfab611a818c0f368af9dea63361ea0fed612/src/actions/transferActions.js#L30